app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -46,8 +49,8 @@
             <% item[:metadata].compact.each do |item_metadata_key, item_metadata_value| %>
               <li class="gem-c-document-list__attribute">
                 <% if item_metadata_key.to_s.eql?("public_updated_at") %>
-                  <time datetime="<%= item_metadata_value.iso8601 %>">
-                    <%= l(item_metadata_value, format: '%e %B %Y') %>
+                  <time datetime="<%= use_month_year_dates ? item_metadata_value.strftime('%Y-%m') : item_metadata_value.iso8601 %>">
```

Comment:
I'd move this logic up to line 15, making a `datetime` variable

Reply:
I thought of doing that initially, but it's a bit awkward because we want to call different methods, so it'd be something like

```ruby
datetime = use_month_year_dates? ->(date) { date.strftime("%Y-%m") } : ->(date) { date.iso8601 }
```

And be called like `datetime(item_metadata_value)`

Which I'm not sure is better

Reply:
Ah I'm with you, didn't spot that it's working on a temporary variable in the loop.